### PR TITLE
Update features and introduce research focus to profile teaser view mode

### PR DESCRIPTION
**Pull request by nlisgo**

### CODE REVIEW

File: src/elife_profile/modules/custom/elife_person_profile/css/person-profile.css [hunk 1]
```diff
@@ -100,15 +100,27 @@
...
     margin: 1em 0;
 }
 
-.person-profile__biog-wrapper .person-profile__biog .person-profile__biog-expertise ul {
+.person-profile__biog-wrapper .person-profile__biog .person-profile__biog-expertise ul, .person-profile__biog-wrapper .person-profile__biog .person-profile__biog-keywords ul {
     padding-left: 0;
     padding-right: 0;
 }
 
+.person-profile__biog-wrapper .person-profile__biog .person-profile__biog-keywords ul li {
```
Comment (davidcmoulton) [inline src/elife_profile/modules/custom/elife_person_profile/css/person-profile.css]: This selector chain and those below have at least 5 levels, which is very deep. Are all these levels required to generate high enough specificity for the rule to apply, or could they be shortened and still give the desired result?

File: src/elife_profile/modules/custom/elife_person_profile/css/person-profile.css [hunk 2]
```diff
@@ -100,15 +100,27 @@
     font-size: 2rem;
 }
 
-.person-profile__biog-expertise {
+.person-profile__biog-expertise, .person-profile__biog-keywords {
     margin: 1em 0;
 }
 
-.person-profile__biog-wrapper .person-profile__biog .person-profile__biog-expertise ul {
+.person-profile__biog-wrapper .person-profile__biog .person-profile__biog-expertise ul, .person-profile__biog-wrapper .person-profile__biog .person-profile__biog-keywords ul {
```
Comment (davidcmoulton) [inline src/elife_profile/modules/custom/elife_person_profile/css/person-profile.css]: Each selector on a new line please.

File: src/elife_profile/modules/custom/elife_person_profile/css/person-profile.css [hunk 3]
```diff
@@ -100,15 +100,27 @@
     font-size: 2rem;
 }
 
-.person-profile__biog-expertise {
+.person-profile__biog-expertise, .person-profile__biog-keywords {
```
Comment (davidcmoulton) [inline src/elife_profile/modules/custom/elife_person_profile/css/person-profile.css]: Each selector on a new line please, so: 
```
.person-profile__biog-expertise,
.person-profile__biog-keywords {
```

**Comment by nlisgo** [comment]
This can be closed. the intent behind this PR was to prepare content on the existing site for migration. I have taken a different approach which means that we don't need to implement this.

**Pull request closed without merging**